### PR TITLE
Redect examples docs

### PR DESCRIPTION
**Pull request by matemax**
update docs

### CODE REVIEW

File: docs/source/sdk/face_detection.rst [hunk 1]
```diff
@@ -23,6 +23,13 @@ You should set detector type when creating detector. Once initialize detector ca
 
     We don’t recommend create often new detector because it is very slowly operation.
 
+Redection
+~~~~~~~~~
+
+If we have only an image and a face bounding box on this image we can run a redect for fast creating the detection
+structure with landmarks. Also you can ran redect if face bounding box belongs to other image but a face shift is small.
+For example you have frames sequence with a face. You can detect face on the first frame and fast redetect this face on
+next frames.
```
Comment (Zeskbest) [inline docs/source/sdk/face_detection.rst]:
```suggestion:-3+0
If there is only one image with a face bounding box on it, you can run the redetect method for a faster face detect (face detection structure creation), with landmarks. 
Also, you can run redetect if the face bounding box belongs to another image and a face shift was small.
For example, you have frame sequence with the same face. You can detect face on the first frame and redetect this face fast on the next frames.
```

File: lunavl/sdk/faceengine/facedetector.py [hunk 2]
```diff
@@ -372,41 +365,34 @@ def redetectOne(self, image, *, bBox=None, detection=None, detect5Landmarks=True
...
+        return None
 
     @CoreExceptionWrap(LunaVLError.DetectFacesError)
     def redetect(
             self,
             images: List[ImageForRedetection],
             detect5Landmarks: bool = True,
             detect68Landmarks: bool = False,
-    ) -> List[FaceDetection]:
+    ) -> List[List[Union[FaceDetection, None]]]:
```
Comment (Zeskbest) [inline lunavl/sdk/faceengine/facedetector.py:395]:
```suggestion
    ) -> List[List[Optional[FaceDetection]]]:
```
https://docs.python.org/3/library/typing.html#typing.Optional

File: lunavl/sdk/faceengine/facedetector.py [hunk 3]
```diff
@@ -421,16 +407,29 @@ def redetect(
             LunaSDKException if an error occurs
         """
 
-        def faceFactory(image: ImageForRedetection) -> Face:
-            face = Face(image.image.coreImage, DetectionFloat(image.bBox.coreRectF, 1.0))
-            return face
+        def faceFactory(image: ImageForRedetection) -> List[Face]:
```
Comment (Zeskbest) [inline lunavl/sdk/faceengine/facedetector.py]: face`s`Factory

**Review by Zeskbest — APPROVED** [review]

File: docs/source/sdk/face_detection.rst (hunk 1 above)
Comment (matemax) [reply docs/source/sdk/face_detection.rst]: +

File: lunavl/sdk/faceengine/facedetector.py (hunk 3 above)
Comment (matemax) [reply lunavl/sdk/faceengine/facedetector.py]: +

**Pull request merged**